package.json

```
@@ -129,6 +129,7 @@
         "tough-cookie": "5.0.0",
         "tsx": "4.19.2",
         "twitter-api-v2": "1.18.1",
+        "ufo": "^1.5.4",
```

```suggestion
        "ufo": "1.5.4",
```
Please pin to the exact version

---

Fixed